entry/src/main/js/default/pages/index/index.js

```diff
@@ -0,0 +1,193 @@
+// @ts-nocheck
+import items from '../../i18n/items.json';
+export default {
+    data: {
+        //      I M A G E S
+        bg: "/common/images/bg.png",
+        wifi: "/common/images/wifi.png",
+        apps: "/common/images/apps.png",
+        lock: "/common/images/lock.png",
+        location: '/common/images/location.png',
+        weather: '/common/images/weather.png',
+        scroll_inactive: '/common/images/scroll_inactive.png',
+        time_bg: "/common/images/time_bg.png",
+        music_bg: "/common/images/music_bg.png",
+        storage_bg: "/common/images/storage_bg.png",
+        tap_to_speak_bg: "/common/images/tap_to_speak_bg.png",
+        refrigerator_doors_bg: "/common/images/refrigerator_doors_bg.png",
+        recepie_bg: "/common/images/recepie_bg.png",
+        bottom_tab_bg: "/common/images/bottom_tab_bg.png",
+        shuffle: '/common/images/shuffle.png',
+        back: '/common/images/back.png',
+        play: '/common/images/play.png',
+        next: '/common/images/next.png',
+        range: '/common/images/range.png',
+        vegetables_range_white: '/common/images/vegetables_range_white.png',
+        green_range: '/common/images/green_range.png',
+        orange_range: '/common/images/orange_range.png',
+        violet_range: '/common/images/violet_range.png',
+        microphone: '/common/images/microphone.png',
+        clock: '/common/images/clock.png',
+        favourite_active: '/common/images/favourite_active.png',
+        favourite_inactive: '/common/images/favourite_inactive.png',
+        home_icon: '/common/images/home.png',
+        storage_icon: '/common/images/storage.png',
+        music_icon: '/common/images/music.png',
+        settings_icon: '/common/images/settings.png',
+        view_icon: '/common/images/view_icon.png',
+        add_icon: '/common/images/add_icon.png',
+        pink_button: '/common/images/pink_button.png',
+
+        //      T E X T
+        storage_settings:'Storage Settings',
+        dinner_text:'Dinner Recepies for you',
+        all_recp_text:'All Recepies',
+        food_remain: 'Food Remaining',
+        home_txt: "Home",
+        storage_txt: "Storage",
+        music_txt: "Music",
+        settings_txt: "Settings",
+        audio_assist1:"How can I help you?",
+        audio_assist2:"Tap to Speak",
+        your_storage:"Your Storage",
+        view_text:"View",
+        add_text:"Add",
+        door_text:"Refrigerator Doors",
+        time:"",
+        day_date:"",
+
+        //      Device Details
+        username: 'UserName',
+        location_city:"Bangalore",
+        //day_date:"Wednesday, 17 Dec",
+        temp_red:"30°",
+
+        //      Music Information
+        music_cover_image:'/common/images/dreams_img.png',
+        song_name: 'Dreams',
+        song_genre: 'Romantic',
+
+        //      Food Remaining tab
+        vegetable_percentage:"72%",
```

Comment:
Similar to getRecipeList, define few more functions and obtain data in similar way - like for food remaining, music info, getDoorStatus etc